src/Apps/Legion/Components/WizardShell/Internal/TwinVerificationStep/Internal/ModelLists.tsx

```diff
@@ -47,10 +47,13 @@ export const ModelLists: React.FC<IModelListsProps> = (_props) => {
         wizardDataManagementContextState.modifiedAssets.properties
     );
     const [models, setModels] = useState<IModelExtended[]>(
-        buildViewModel(
-            wizardDataManagementContextState.initialAssets.models,
-            wizardDataManagementContextState.modifiedAssets.models
-        )
+        wizardDataManagementContextState.initialAssets?.models &&
```

Comment:
can we just handle null checks inside of `buildViewModel` instead and return empty array id initial assets and modified assets are null or something?

Reply:
There is a bug that causes initalAssets and ModifiedAssets to be null, that is why it was failing. I'll take a look at it in the next PR since I am refactoring ModelLists completely for Modify step.